core/mantle_upgrade.go

```diff
@@ -11,34 +11,53 @@ var (
 		ChainID:               params.MantleMainnetChainId,
 		BaseFeeTime:           u64Ptr(0),
 		BVMETHMintUpgradeTime: u64Ptr(0),
-		MetaTxUpgradeTime:     u64Ptr(0),
+		MetaTxV2UpgradeTime:   u64Ptr(0),
+		MetaTxV3UpgradeTime:   u64Ptr(0), //TODO set upgrade timestamp
 	}
 
 	MantleSepoliaUpgradeConfig = MantleUpgradeChainConfig{
 		ChainID:               params.MantleSepoliaChainId,
 		BaseFeeTime:           u64Ptr(1_704_891_600),
 		BVMETHMintUpgradeTime: nil, //TODO set upgrade timestamp
-		MetaTxUpgradeTime:     nil, //TODO set upgrade timestamp
+		MetaTxV2UpgradeTime:   nil, //TODO set upgrade timestamp
+		MetaTxV3UpgradeTime:   nil, //TODO set upgrade timestamp
+	}
+	MantleSepoliaQA3UpgradeConfig = MantleUpgradeChainConfig{
+		ChainID:               params.MantleSepoliaQA3ChainId,
+		BaseFeeTime:           u64Ptr(0),
+		BVMETHMintUpgradeTime: u64Ptr(0),
+		MetaTxV2UpgradeTime:   u64Ptr(0),
+		MetaTxV3UpgradeTime:   u64Ptr(1717689600),
```

Comment:
Is this timestamp suitable for QA3 network?

Reply:
yes, the upgrade of QA3 has been completed